LocalStorage.coffee

```diff
@@ -124,6 +124,19 @@ class LocalStorage extends events.EventEmitter
         if e.code != "EEXIST"
           throw e
       return
+  sync:()->
+    @_bytesInUse = 0;
```

Comment:
Since this is a cut-and-paste of the section from `_init`, please pull that code out of there and call this to achieve DRY.

Reply:
> Since this is a cut-and-paste of the section from `_init`, please pull that code out of there and call this to achieve DRY.

Would it be simpler to stick `_sync` into `_init`? If you are not happy with the name I can put the piece of code into a function called `toAchieveDry` and call it from sync and init.

Reply:
this [commit](https://github.com/lmaccherone/node-localstorage/pull/79/commits/1c2f78e8921316eca7b45660bdf48df6eb7c7300) should address this